decidim-elections/app/presenters/decidim/elections/admin_log/election_presenter.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    module AdminLog
+      # This class holds the logic to present a `Decidim::Election`
+      # for the `AdminLog` log.
+      #
+      # Usage should be automatic and you shouldn't need to call this class
+      # directly, but here's an example:
+      #
+      #    action_log = Decidim::ActionLog.last
+      #    view_helpers # => this comes from the views
+      #    ElectionPresenter.new(action_log, view_helpers).present
+      class ElectionPresenter < Decidim::Log::BasePresenter
+        private
+
+        def diff_fields_mapping
+          {
+            name: :i18n,
+            published_at: :date,
+            weight: :integer
+          }
+        end
+
+        def i18n_labels_scope
+          "activemodel.attributes.election"
+        end
+
+        def action_string
+          case action
+          when "publish", "unpublish"
+            "decidim.elections.admin_log.election.#{action}"
```

Comment:
The keys must be static for the `i18n` check to detect the translations used, such as:
 
```suggestion
          when "publish"
            "decidim.elections.admin_log.election.publish"
          when "unpublish"
            "decidim.elections.admin_log.election.unpublish"
```

💁‍♂️

Reply:
I tried using the static form and deleting `- decidim.elections.admin_log.*` from `config/i18n-tasks.yml`, but it gives again the RSpec error:

https://github.com/decidim/decidim/pull/6293/checks?check_run_id=882515123

Reply:
Oooh 😪, excuse me.

Reply:
no problem!! :)